apps/arweave/src/ar_data_discovery.erl

```diff
@@ -204,9 +205,8 @@ pick_peers(Peers, PeerLen, N) ->
 
 collect_peers() ->
 	N = ?DATA_DISCOVERY_COLLECT_PEERS_COUNT,
-	%% rank peers by their current rating since we care about their recent throughput performance
-	collect_peers(lists:sublist(ar_peers:get_peers(current), N)).
-
+	%% rank peers by their current rating since we care out their recent throughput performance
+	collect_peers(lists:sublist(ar_peers:get_peers(?DEFAULT_PEERS_LIST_FILTER), N)).
```

Comment:
Adding to my comment above, I'd suggest we have distinct peers lists that we use everywhere similar to what we hadd before. E.g. `get_peers(lifetime)` and `get_peers(current)` and then use the a constant or a config value when building those lists (rather than passing the value in when making the query)

the idea is to limit the number of peer lists for now and not open up the possibility for different areas of code to filter based on different day amounts (atleast not add that flexibility until we need it)

Also if you want to rename the peers lists that's fine too. `lifetime` and `current` aren't totally right any more. 

`lifetime` is probably more accurately named something like `consensus` or `trust`
and `current` maybe more accurately named something like `performance` (although `current` does also kinda work)